common/views/components/CataloguePageLayout/CataloguePageLayout.tsx

```diff
@@ -1,15 +1,17 @@
 import { FunctionComponent, useEffect, useState } from 'react';
 import PageLayout, { Props as PageLayoutProps } from '../PageLayout/PageLayout';
 import InfoBanner from '../InfoBanner/InfoBanner';
-import SignIn from '@weco/catalogue/components/SignIn/SignIn';
+import SignIn from '../SignIn/SignIn';
 import { prefix } from '@weco/identity/src/utility/prefix';
 
 type Props = {
   hideInfoBar?: boolean;
+  hideSignIn?: boolean;
 } & PageLayoutProps;
 
 const CataloguePageLayout: FunctionComponent<Props> = ({
   hideInfoBar,
+  hideSignIn,
```

Comment:
If it worth making this 1 prop about `hideTopContent` so anything in this position isn't shown?

Reply:
Good idea, I'll do that